projects/igniteui-angular/src/lib/tabs/tab-item.component.ts

```diff
@@ -73,6 +73,10 @@ export class IgxTabItemComponent extends IgxTabItemBase implements AfterViewInit
         this._label = newValue;
     }
 
+    /**@hidden*/
+    @Input()
+    public autoGenerated: boolean;
+
```

Comment:
While this prop seems like private for the component, please document what it does.